mods/bucket/init.lua

```diff
@@ -46,6 +46,10 @@ function bucket.register_liquid(source, flowing, itemname, inventory_image, name
 	bucket.liquids[flowing] = bucket.liquids[source]
 
 	if itemname ~= nil then
+		if groups and groups.overlay_texture then
+			inventory_image = "bucket_overlay.png^"..inventory_image.."^[noalpha^bucket_overlay.png^[makealpha:255,126,126"
```

Comment:
unless i am wrong, the first `bucket_overlay.png` would get fully overlay/cover by the inventory_image so there is no reason for it.